### PR TITLE
Allow to delete non-migrated previews instead of moving them

### LINKED ISSUE #30167: Delete non-migrated previews for regeneration
### Steps
1. have an old instance with 100k previews
2. upgrade to 20
3. see you have to manually migrate those old previews
4. see it takes you 100+ days to finish

### Solution
 Delete those non-migrated previews as it's faster to generate those on demand than move them
Especially between buckets.

## Context
https://github.com/nextcloud/server/blob/215aef3cbdc1963be1bb6bca5218ee0a4b7f1665/core/Command/Preview/Repair.php

### PR DESCRIPTION
**Pull request by skjnldsv**
fix #30167

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Review by come-nc — APPROVED** [review]

File: core/Command/Preview/Repair.php [hunk 1]
```diff
@@ -94,10 +95,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
 
 		$dryMode = $input->getOption('dry');
+		$deleteMode = $input->getOption('delete');
+
 
 		if ($dryMode) {
 			$output->writeln("INFO: The migration is run in dry mode and will not modify anything.");
 			$output->writeln("");
+		} elseif ($deleteMode) {
+			$output->writeln("WARN: The migration will _DELETE_ old previews.");
```
Comment (PVince81) [inline core/Command/Preview/Repair.php:105]: it won't in dry mode though ?

Comment (skjnldsv) [reply core/Command/Preview/Repair.php:105]: this line is only displayed when drymode is not true

**Pull request merged**